tpot/tpot.py

```diff
@@ -790,6 +792,51 @@ def _gradient_boosting(self, input_df, learning_rate, max_features, min_weight):
             learning_rate=learning_rate, n_estimators=500,
             max_features=max_features, random_state=42, min_weight_fraction_leaf=min_weight)
 
+    def _mdr(self, input_df, tie_break, default_label, num_features_to_combined = 2, score = None):
+        """Fits the Multi-dimensionality Reduction Classifier/Feature Constructor
+
+        Parameters
+        ----------
+        input_df: pandas.DataFrame {n_samples, n_features+['class', 'group', 'guess']}
+            Input DataFrame for fitting the MDR feature constructor
+        tie_break: int
+            One of the class values (currently categorical) that determines which class to assign to the combined feature instance, in case class ratio among 
+        possible classes of that feature instance vector is the same as the class ratio in the general population.
+            Maximum number of features to use (proportion of total features)
+        default_label: int
+            One of the class values (currently categorical) that assigns a class label to combine feature instances that the algorithm has not seen before.
+        num_features_to_combined: int
```

Comment:
Don't have a default for num_features_to_be_combined; TPOT should provide this value.